docs/src/cli/install.md

```diff
@@ -154,7 +154,7 @@ installed on your system.
 
 ### Prerequisites
 
-Before building from source, make sure to install the following prerequisites:
+Before building from source, or developing Solana programs, make sure to install the following prerequisites:
```

Comment:
With the added bit about developing programs, this sentence sounds a little "off" to me now. What do you think about flipping it around:
```
Install the following prerequisites before building from source or developing Solana programs:
```
Looking in this file, I see several other sentences that start with "Install Y` instead of `Before X, install Y` so I think my suggestion would make this line more consistent with the overall doc

Reply:
works for me